randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures in this room are "
+            "some of the few that cannot be destroyed by Power Bombs.\nNote that some game versions change that."
+        ),
+    ],
+    "1618265307": [
+        # Burn Dome PB Sandstone Tip
+        # Replaces Burn Dome sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "745991660": [
+        # Gathering Hall PB talloric alloy Tip
+        # Replaces Gathering Hall lamp scans
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Talloric Alloy&main-color=#89D6FF; structures, "
+            "the grate above can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3344095433": [
+        # Ruined Shrine PB Sandstone Tip
+        # Replaces Ruined Shrine sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3485565452": [
+        # Vault PB Slots Tip
+        # Replaces Vault bomb slot seal scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;These Bomb Slot covers are able to be destroyed by &main-color=#FF3333;Power Bombs"
+            "&main-color=#89D6FF;.\n\nAdditionally, these Bomb Slots are also able to be activated by Power Bombs."
+        ),
+    ],
+    "1354393071": [
+        # Watery Hall PB Sandstone Tip
+        # Replaces Watery Hall sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "3507547160": [
+        # Lava Lake PB Sandstone + No-Respawn Tip
+        # Replaces Lava Lake sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Like most &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures, "
+            "this can also be destroyed by &main-color=#FF3333;Power Bombs&main-color=#89D6FF;.\n\n"
+            "Additionally, in most game versions, these rocks will not reappear after being destroyed."
```

Comment:
I do not recall any game versions having this change. Randomprime does have a QOL change that makes it so that they do not reappear, which may be what you are thinking of.

Reply:
Like the other Game Versions comment, this was added because (IIRC) it doesn't work that way in AP.

Reply:
this doesn't work in vanilla / legacy mode. because this is as mentioned by sam, a randomprime QoL change.